.github/workflows/build-docs.yml

```diff
@@ -0,0 +1,32 @@
+name: Build Docs
+on: push
+      
+permissions:
+  contents: write
+  
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout 🛎️
+        uses: actions/checkout@v4
+        with:
+          persist-credentials: false
+
+      - name: Setup Miniconda
+        uses: conda-incubator/setup-miniconda@v2
+        with:
+          auto-update-conda: true
+          auto-activate-base: false
+          miniconda-version: 'latest'
```

Comment:
why are we installing `conda` at all? Further down we use `pip` to install things...
Is this a relic of some other project or is `conda` really needed for something?

Reply:
you're right, this was carried over from another project. Removing...